### ISSUE TITLE
ng-change event doesn't get triggered when using uif-searchbox

### ISSUE THREAD

**Issue by bhosalemahesh899** [ngOfficeUIFabric/ng-officeuifabric#488]
Hi,

I am using uif-searchbox and I want to do something on ng-change of uif-searchbox.
but the ng-change event doesn't get triggered on change
ng-cahnge event getting called on click on close button of uif-searchbox

 <uif-searchbox value="vm.searchTerm" ng-model="vm.searchTerm" ng-change="vm.onSearch()" />

please see in the following js
https://jsfiddle.net/bhosalemahesh899gmailcom/owv9fw4f/27/

**Comment by andrewconnell** [comment]
Correct... ATM this control like many others does not emit this event. It does use it internally to change the `$isDirty` flag on the component. Therefore this would be an enhancement.
